src/yandex_cloud_ml_sdk/_datasets/dataset.py

```diff
@@ -161,14 +166,41 @@ async def _download(
         return await asyncio.wait_for(self.__download_impl(
             base_path=base_path,
             exist_ok=exist_ok,
+            timeout=timeout,
```

Comment:
Why we need to pass this timeout here?
This is global download timeout, we don't need it in internal calls